javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
-  }
+  
+    this intervalID = setInterval(() => {
+      this.currentTime = intervalCounter += 1;
+      
+      if(callback){
+      return callback;
+      }
+
+    }, 1000);
+   
+   
+    }
+}
 
   getMinutes() {
```

Comment:
all the methods under line 21 are out of the class. the curly brace in line 20 closes the class